src/annotator/test/annotation-sync-test.js

```diff
@@ -45,9 +45,9 @@ describe('AnnotationSync', function() {
     sandbox.restore();
   });
 
-  describe('channel event handlers', function() {
-    describe('the "deleteAnnotation" event', function() {
-      it('broadcasts the "annotationDeleted" event over the local event bus', function() {
+  describe('#constructor', function() {
+    context('when "deleteAnnotation" is published', function() {
```

Comment:
The problem with organizing the tests for `AnnotationSync` is that `AnnotationSync` is mostly just one huge constructor function (along with several private helper functions that are called only by the constructor) and the main thing that constructor does is register event handlers.

Given that it's the constructor that registers the event handlers, I grouped the tests for what happens when the events are called under `describe('#constructor'` but I further grouped the tests for each event in a `context()`.